__tests__/app.test.js

```diff
@@ -312,4 +312,117 @@ describe("GET comments by review ID", () => {
                 });
         });
     });
+});
+
+describe("POST new comment", () => {
+    describe("Happy path :)", () => {
+        let testComment = {
+            username: "philippaclaire9",
+            body: "Where is my head?"
+        };
+
+        test("When a valid comment is added it is returned as an object", () => {
+            return request(app)
+                .post("/api/reviews/1/comments")
+                .send(testComment)
+                .expect(201)
+                .then(({ body }) => {
+                    const newComment = body.comment;
+                    expect(newComment.author).toEqual(testComment.username);
+                });
+        });
+
+        test("Ensure that only the posted comment is returned", () => {
+            return request(app)
+                .post("/api/reviews/2/comments")
+                .send(testComment)
+                .expect(201)
+                .then(({ body }) => {
+                    const newComment = body.comment;
+                    expect(newComment.author).toEqual(testComment.username);
+                });
```

Comment:
this test (line 335) is identical to the test on line 324 - I would delete it as it isn't adding any behaviour to this endpoint